docs/en/rules/Azure.ACR.ExportPolicy.md

```diff
@@ -0,0 +1,119 @@
+---
+reviewed: 2025-07-09
+severity: Important
+pillar: Security
+category: DP:02 Data Protection
+resource: Container Registry
+resourceType: Microsoft.ContainerRegistry/registries
+online version: https://azure.github.io/PSRule.Rules.Azure/en/rules/Azure.ACR.ExportPolicy/
+---
+
+# Container Registry export policy should be disabled
+
+## SYNOPSIS
+
+Disable export of artifacts from Azure container registry to ensure data is accessed solely via the data plane.
+
+## DESCRIPTION
+
+Azure Container Registry (ACR) export policy allows copying container images and artifacts to other registries or locations. 
+When the export policy is enabled, data can be moved out of the registry via 'acr import' or 'acr transfer' commands.
+
+To improve security and prevent data exfiltration, the export policy should be disabled.
+This ensures that data in the registry is accessed solely through the data plane using standard Docker commands like 'docker pull'.
+
+Disabling export policy requires that public network access is also disabled.
+This provides additional protection by ensuring the registry is only accessible from private networks.
+
+## RECOMMENDATION
+
+Consider disabling the export policy and public network access for container registries containing sensitive data.
+
+## EXAMPLES
+
+### Configure with Bicep
+
+To deploy registries that pass this rule:
+
+- Set `properties.policies.exportPolicy.status` to `disabled`.
+- Set `properties.publicNetworkAccess` to `Disabled`.
```

Comment:
Remove setting `properties.publicNetworkAccess` to `Disabled` for both Azure template and Bicep examples.